test/unit/org/apache/cassandra/index/sai/SAITester.java

```diff
@@ -199,6 +419,167 @@ protected ObjectName objectNameNoIndex(String name, String keyspace, String tabl
         }
     }
 
+    protected ObjectName bufferSpaceObjectName(String name) throws MalformedObjectNameException
+    {
+        return new ObjectName(String.format("org.apache.cassandra.metrics:type=StorageAttachedIndex,name=%s", name));
+    }
+
+    protected long getSegmentBufferSpaceLimit() throws Exception
+    {
+        ObjectName limitBytesName = bufferSpaceObjectName("SegmentBufferSpaceLimitBytes");
+        return (long) (Long) getMetricValue(limitBytesName);
+    }
+
+    protected Object getSegmentBufferUsedBytes() throws Exception
+    {
+        ObjectName usedBytesName = bufferSpaceObjectName("SegmentBufferSpaceUsedBytes");
+        return getMetricValue(usedBytesName);
+    }
+
+    protected Object getColumnIndexBuildsInProgress() throws Exception
+    {
+        ObjectName buildersInProgressName = bufferSpaceObjectName("ColumnIndexBuildsInProgress");
+        return getMetricValue(buildersInProgressName);
+    }
+
+    protected void upgradeSSTables()
+    {
+        try
+        {
+            StorageService.instance.upgradeSSTables(KEYSPACE, false, currentTable());
+        }
+        catch (Throwable e)
+        {
+            throw new RuntimeException(e);
+        }
+    }
+
+    protected long totalDiskSpaceUsed()
+    {
+        ColumnFamilyStore cfs = Keyspace.open(KEYSPACE).getColumnFamilyStore(currentTable());
```

Comment:
This and another 14 usages across this class can be slightly simplified by just using the inherited `CQLTester#getCurrentColumnFamilyStore()`.

Reply:
We might also benefit from a similar utility method to get the current index group, something like:
```java
protected StorageAttachedIndexGroup getCurrentIndexGroup()
{
    return StorageAttachedIndexGroup.getIndexGroup(getCurrentColumnFamilyStore());
}
```